### DISCUSSION TITLE
本地测试时图片地址不正确，请问我是需要修改哪部分配置文件？

### DISCUSSION THREAD

**Discussion by xiaonuoz** [CaiJimmy/hugo-theme-stack#744] [Q&A] [answered]
在vscode中查看图片是找到了的。
![image](https://user-images.githubusercontent.com/46923935/208396392-1f9a1528-ecec-457a-8a87-c17da8c75056.png)
运行hugo.exe server -D 访问
![image](https://user-images.githubusercontent.com/46923935/208396561-3ea6a3d7-2583-4ded-a6cb-86275b8f61c0.png)
使用hugo.exe 也没有在public中生成对应的图片，请问它的地址对应的是哪部分的配置文件？

**Comment by zhixuan2333** [comment, marked as answer]
是不是在写path的时候是加根号了啊？
``` md
❌ [image](/Your_Image_Path.png)
✔️ [image](Your_Image_Path.png)
```

**Reply by xiaonuoz** [reply]
![image](https://user-images.githubusercontent.com/46923935/208561827-1ffeee1f-025f-42ca-8f09-e3c683cb1c2f.png)
没有啊，我根据测试模板来的，可是测试模板生成了图片，我这个没生成
![image](https://user-images.githubusercontent.com/46923935/208562094-4b56b234-c953-4176-99cc-aa31956ef5ae.png)

md文件是需要设置什么特别的东西吗

**Reply by xiaonuoz** [reply]
找到问题了，md的文件名必须是index.zh-cn，我没注意到这个
